README.md

```diff
@@ -239,9 +239,9 @@ You just need the below `hello_world_status_ok_assertions.json`.
 
 the `github_host.properties` looks as below:
 ```
-restful.application.endpoint.host=https://api.github.com
-restful.application.endpoint.port=443
-restful.application.endpoint.context=
+web.application.endpoint.host=https://api.github.com
```

Comment:
👍

Reply:
code changes updated for backword compatibility